src/vorta/network_status/darwin.py

```diff
@@ -43,14 +43,14 @@ def get_known_wifis(self):
         result = []
         if wifis is not None:
             for wifi in wifis.values():
-                raw_last_connected = wifi.get('LastConnected', None)
-                last_connected = None if not raw_last_connected \
-                    else format_date_time(raw_last_connected, WifiSettingModel.last_connected.formats)
                 ssid = wifi.get('SSIDString', None)
-
                 if ssid is None:
                     continue
 
+                last_connected = wifi.get('LastConnected', None) or wifi.get('LastAutoJoinAt', None)
+                if isinstance(last_connected, str):  # TODO: Maybe not needed any more?
```

Comment:
Could this become `if last_connected is not None:`?

Reply:
No, because if it's `None` we can just save it to the DB.

There seem to be cases where `plistlib` returns a string, rather than a datetime object. Maybe for older versions. So we convert the string to datetime, as the original author did. For me it always returned a datetime object, but older macOS versions may use a different version of this file. So we keep it for now.

If `last_connected` is None, we don't want to convert the string to datetime, but just save the None as-is.

So we want either a None or a datetime, but never a string.